app/code/Magento/Sales/Model/Order/Creditmemo.php

```diff
@@ -379,6 +389,9 @@ public function canRefund()
      */
     public function getInvoice()
     {
+        if (!$this->getData('invoice') instanceof \Magento\Sales\Model\Order\Invoice && $this->getInvoiceId()) {
```

Comment:
Is there a reason not to use `\Magento\Sales\Api\Data\InvoiceInterface` here?

Reply:
Good point. No there is no reason.